.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,249 @@
+      with:
+        repository: nspcc-dev/neo-go
+        ref: ${{ github.event.inputs.neo_go_branch || 'v0.101.0' }}
+        path: neo-go
+
+    # neofs-contract
+    - name: Checkout neofs-contract repository
+      uses: actions/checkout@v3
+      with:
+        repository: nspcc-dev/neofs-contract
+        ref: ${{ github.event.inputs.neofs_contract_branch || 'v0.17.0' }}
+        path: neofs-contract
+
+    # neofs-locode-db
+    - name: Checkout neofs-locode-db repository
+      uses: actions/checkout@v3
+      with:
+        repository: nspcc-dev/neofs-locode-db
+        ref: ${{ github.event.inputs.neofs_locode_db_branch || 'v0.3.0' }}
+        path: neofs-locode-db
+
+###############################################################################
+###############################################################################
+###############################################################################
+# Prerequisites
+    # Set up Go
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.18.2
+
+###############################################################################
+###############################################################################
+###############################################################################
+    # Build and install NEOFS parts on runner
+    # Install neofs-cli
+    - name: Install neofs-node (neofs-cli and neofs-adm)
+      run: |
+        make all
+        sudo cp bin/neofs-cli /usr/local/bin/neofs-cli
+        sudo cp bin/neofs-adm /usr/local/bin/neofs-adm
+        cp bin/neofs-cli /tmp/neofs-cli
+        cp bin/neofs-adm /tmp/neofs-adm
+      working-directory: neofs-node
+
+    # Install neofs-s3-authmate
+    - name: Install neofs-s3-authmate
+      run: |
+        make
+        sudo cp bin/neofs-s3-authmate /usr/local/bin/neofs-s3-authmate
+      working-directory: neofs-s3-gw
+
+    # Install neo-go
+    - name: Install neo-go
+      run: |
+        make
+        sudo cp bin/neo-go /usr/local/bin/neo-go
+      working-directory: neo-go
+
+    # Install neofs-contract
+    - name: Install neofs-contract
+      run: |
+        NEOGO=../neo-go/bin/neo-go make all
+        mkdir -p ${{ env.CONTRACT_DIR }}
+        cp -r ./* ${{ env.CONTRACT_DIR }}
+      working-directory: neofs-contract
+
+    # Install neofs-locode-db
+    - name: Install neofs-locode-db
+      run: |
+        echo "Using v0.3.0"
+      working-directory: neofs-locode-db
+
+###############################################################################
+###############################################################################
+###############################################################################
+    # Transfer the env file for the NEOFS test cluster deployment to docker
+    - name: Copy .env file to .env in neofs-dev-env
+      run: |
+        cp .env ${GITHUB_WORKSPACE}/neofs-dev-env/.env
+        cat .env
+        cat ${GITHUB_WORKSPACE}/neofs-dev-env/.env
```

Comment:
So we have v0.36.0 tested as per env configuration? I'd expect a target reference being used, usually it's some commit hash.